lib/Avatar/AvatarWithInformation.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { Avatar } from '../';
+
+class AvatarWithInformation extends Component {
```

Comment:
I'm not too sure if we need to create a new component for showing information within the Avatar component. `AvatarWithPopover` requires some funky logic to get working but I believe this could be a simple case of rendering children into the `Avatar` component. e.g.

```
<Avatar>
  <AvatarInformation />
</Avatar>
```

Obviously this will have an affect on how the styling is currently setup and maybe we could add the `avatar__wrapper` in another way but I believe it'd be better if we didn't add another component for rendering additional content for the Avatar component itself.

The main reason is it becomes harder to discover where stuff happens but keeping it in one component with stuff like this is the best approach. When a component gets too large then that is a different issue but `Avatar` is pretty lean right now.